src/module/helpers.ts

```diff
@@ -102,14 +112,13 @@ export class Helpers {
      *
      * @param value
      * @param options Define the range the given value must be in (or none)
+     * @returns True if the value was modified, false otherwise
      */
-    static applyValueRange(value: number, options?: CalcTotalOptions): number {
-        if (typeof options?.min === 'number') {
+    static applyRange(value: number, options?: CalcTotalOptions) {
+        if (options?.min != null)
```

Comment:
There were cases of ModifiableValue using string (damage type). Those also were modified by effect changes in the past.

Did you consider those?

Reply:
`ModifiableValue` only accept `number` as its type is not flexible. I remember we've talked about that. It might be a bug since I might have removed it for not understanding at that time. Trying to find our conversation about this point.